underfs/ozone/pom.xml

```diff
@@ -25,18 +25,25 @@
     <!-- The following paths need to be defined here as well as in the parent pom so that mvn can -->
     <!-- run properly from sub-project directories -->
     <build.path>${project.parent.parent.basedir}/build</build.path>
+    <ufs.ozone.version>1.0.0</ufs.ozone.version>
   </properties>
 
+  <profiles>
```

Comment:
do we need this? this looks to run by default, do we currently support other way to build without this profile?
If not, we should simply have it in dependency rather than being selected by a profile

Reply:
ok